### ISSUE TITLE
Black screen in 0.2.15

### ISSUE THREAD

**Issue by saddy001** [neutrinolabs/xorgxrdp#180]
Hello, this is 
* Ubuntu 18.04.5 with compiled xrdp v0.9.15 and xorgxrdp v0.2.15
* ./configure --enable-fuse --enable-pixman --enable-tjpeg

While version .14 was all fine, after updating to v.15, everything seemed fine here too. Only after about 24hours or so, I'm getting a black screen after login. From then on, this is permanent. The server does not much in that time, but two things: starting a virtualbox VM and hybernating "/usr/sbin/rtcwake -m off -s 36000". 
I narrowed down working vs non-working logs to two essential lines in /home/user/.xorgxrdp.*.log. In the non-working log situation, I'm getting lines 
```
jpeg support 8807
rdpCapture3: unimplemented color conversion
```
while in the working versions, I get 
`jpeg support 0`

I believe jpeg support returns an error code here, being 8807 in the error case and 0 in the no-error case.
```
xorgxrdp/module/rdpClientCon.c:
843     LLOGLN(0, ("  jpeg support %d", clientCon->client_info.jpeg));
```
If I use another client (Opera instead of Firefox) the issue persists. The other logs (/var/log/xrdp.log, /var/log/xrdp-sesman.log) contain no relevant difference between working and non-working situations, but if you need them, I have them here. Also rebooting does not help.

I believe there is some issue in the clientCon->client_info function.

**Comment by saddy001** [comment]
It also happens in v0.2.14.

**Comment by metalefty** [comment]
Can you confirm what happens if you turn on JPEG on the CLIENT?

**Comment by saddy001** [comment]
Thanks for your reply. I'm using apache guacamole's web based client. As far as I know, there is no option to enable JPEG. The current status is the following:

* v.14 seems also affected
* I removed all config files of x*rdp
* I switched from checkinstall to make install
* I reverted to v.14, mostly because there seem to be some errors in the log, that are not really errors (occur also when all is okay, there is already a ticket for that)
* I checked the logs again in the last black-screen situation and this time, I found no suspicious jpeg support lines. Moreover I found nothing that looks like a real error. 

Normally I would paste all the logs now, but this time, I threw them away (it was late). I suggest I'll migrate to v.16 when it's ready and open a new issue with logs if it happens again. Except you need any more information?

**Comment by metalefty** [comment]
Thanks for the feedback. BTW, I meant you to try turn on/off JPEG. That was my typo. Anyway, I'll have a look. Thanks!

**Comment by saddy001** [comment]
It's stable again. My guess is that it was one of the config files that broke things. Although I never changed one of them.

`rm -R /etc/xrdp /etc/X11/xrdp /etc/pam.d/xrdp-sesman /usr/sbin/xrdp* /var/log/xrdp*`
